helm/solr-operator/crds/crds.yaml

```diff
@@ -1053,9 +1053,12 @@ spec:
                 type: object
               repositoryName:
                 description: The name of the repository to use for the backup.  Defaults to "legacy_local_repository" if not specified (the auto-configured repository for legacy singleton volumes).
+                pattern: '[a-zA-Z0-9]([-_a-zA-Z0-9]*[a-zA-Z0-9])?'
                 type: string
               solrCloud:
                 description: A reference to the SolrCloud to create a backup for
+                minLength: 63
+                pattern: '[a-z0-9]([-a-z0-9]*[a-z0-9])?'
```

Comment:
[-1/Q] Ditto to my reminder comment above, but this time regarding the lower-case requirement on this string field that you may or may not have intended.

Reply:
So yeah, just to be very clear, I'm just following the guidelines for [kubernetes resource names](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-label-names). This field is a reference to SolrCloud resources, so there is no way that giving a string that has a length > 63, or doesn't match that regex, is going to be successful (because we will never be able to find a SolrCloud in kubernetes with the given name).